### ISSUE TITLE
web-mode support

### ISSUE THREAD

**Issue by itamarst** [jscheid/dtrt-indent#28]
web-mode.org looks pretty neat, it'd be great if I could get it to not have terrible indentation.

It supports multiple languages with different levels, so probably you want multiple lines added to `dtrt-indent-hook-mapping-list` (and I don't know if this will work):

```
    (web-mode-prog-mode        javascript    web-mode-code-indent-offset) ;
    (web-mode-prog-mode        css                  web-mode-css-indent-offset) ;
    (web-mode-prog-mode        sgml              web-mode-markup-indent-offset) ;
```

Also it does its own funky thing involving tabs (have to call `(web-mode-use-tabs)` or `(web-mode-use-spaces)`, and hopefully that is buffer specific and not global), so you might need to add an extension mechanism for tabs similar to that for indents.

**Comment by rrthomas** [comment]
Sounds good, if you can come up with a patch I'd certainly be interested.

**Comment by rrthomas** [comment]
Closed by PR #60.
